validate.py

```diff
@@ -4,10 +4,24 @@
 """
 import sys
 import ljmmm
+import traceback
+import os
```

Comment:
Same thing as `save_changes.py`, please clean up the os module import.

Reply:
I agree.